python/tvm/relay/op/_tensor_grad.py

```diff
@@ -803,3 +805,15 @@ def arange_grad(orig, grad):
     grad_step = cast_like(_sum(grad_step), step)
 
     return [grad_start, grad_stop, grad_step]
+
+
+@register_gradient("gather_nd")
+def gather_nd_grad(orig, grad):
+    data, indices = orig.args
+    return [scatter_nd(grad, indices, data.checked_type.concrete_shape), zeros_like(indices)]
+
+
+# @register_gradient("scatter_nd")
```

Comment:
uncomment or remove, is something wrong with it?

Reply:
I'm getting some really weird results when using gather_nd as the gradient. Values are like 1e303.